agnpy/absorption/tests/test_absorption.py

```diff
@@ -228,6 +228,49 @@ def test_abs_dt_vs_point_source(self):
         # requires a 10% deviation from the two SED points
         assert check_deviation(nu, tau_dt, tau_ps_dt, 0.1)
 
+    def test_abs_blr_cubepy(self):
```

Comment:
now that you added test_tau_accuracy_standard_vs_cubepy this test does not seem to add anything to it, so I think you can remove test_abs_blr_cubepy